### ISSUE TITLE
Blank Screen

### ISSUE THREAD

**Issue by alexandrevicenzi** [flathub/com.skype.Client#8]
Skype doesn't load anything, it's just a blank screen on openSuse Leap 42.3.

![image](https://user-images.githubusercontent.com/2637672/33050856-3de21de4-ce4e-11e7-85fa-3703a1bbcda2.png)

**Comment by marcosps** [comment]
Using Fedora 27, it works perfectly.
Flatpak version: Flatpak 0.10.0

@alexandrevicenzi you mentioned that it does not work even if you grab the official skype and run it outside of flatpak, right?

**Comment by alexandrevicenzi** [comment]
Yeah, looks like the official version looks the same. Not sure how to debug this problem.

**Comment by alexandrevicenzi** [comment]
Well, after rebooting the pc it's working. Have no idea why.

**Comment by alexandrevicenzi** [comment]
Official Skype works, version from flatpak don't. There's no error in the logs.
